atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -319,3 +289,47 @@ def run(self):
         if self.reserve_build and not is_scratch_build(self.workflow):
             self.reserve_build_in_koji(component, version, release, release_label,
                                        dockerfile_labels)
+
+    def _get_nvr(self, dockerfile_labels) -> Tuple[str, str, Optional[str]]:
+        """Get the component, version and release labels from the Dockerfile."""
+        labels = Labels(dockerfile_labels)
+
+        component_label = labels.get_name(Labels.LABEL_TYPE_COMPONENT)
+        component: Optional[str] = dockerfile_labels.get(component_label)
+
+        version_label = labels.get_name(Labels.LABEL_TYPE_VERSION)
+        version: Optional[str] = dockerfile_labels.get(version_label)
+
+        release_label = labels.get_name(Labels.LABEL_TYPE_RELEASE)
+        release: Optional[str] = dockerfile_labels.get(release_label)
+
+        missing_labels = {}
+
+        # component: missing X, empty OK
+        if component is None:
+            self.log.error("missing label: %s", component_label)
+            missing_labels[component_label] = "missing"
```

Comment:
because it would fail already in osbs-client here : https://github.com/containerbuildsystem/osbs-client/blob/master/osbs/api.py#L111

Reply:
Hmm, then my follow-up question would be, why is empty version not OK? :smile: 

I'll just update the code to handle them all the same way (except release, which doesn't have to be set)

Reply:
the o-c checks only existence of 'version' not the emptiness (as it may be using variable), here we are already checking with inheritance so version can't be empty